packages/core/src/fhir-to-cda/composition-splitter.ts

```diff
@@ -26,13 +26,13 @@ export function splitBundleByCompositions(fhirBundle: Bundle): Bundle[] {
       ? findResourceInBundle(fhirBundle, patientReference)
       : undefined;
     if (!patientResource) {
-      throw new BadRequestError("Patient resource not found");
+      throw new BadRequestError("Invalid Patient reference in the subject field");
```

Comment:
For consistency, shouldn't "subject" here have capital first letter, like "Patient" does? Here and below.

Reply:
No, I'm matching the casing to how it appears in FHIR Bundles. FHIR Resources are always capitalized, but their attributes are lowercased.

Reply:
To add to this - put yourself on the receiving the of the error, it could be more helpful.

The reference could very well be valid, but it's just the resource that's missing in the bundle, right? Perhaps:

`Could not find resource corresponding to the Patient reference in the subject field in the provided bundle`

Reply:
Tbh, because we will be adding the resources for both, the Patient and the Organization, this error will never happen, so it serves more like a safeguard to ensure the patientResource exists for the context of this function. 

As for Organization in the next line, the error would be the result of a faulty reference specifically.

Reply:
Are you going to take Dima's suggestion on the message? Here and below (Org)

Reply:
No. As I explained, we are providing these resources automatically, so it's a message to us.